### PR TITLE
Show tag-targeted logos on tag pages

### PR DESCRIPTION
**Pull request by kelvin-chappell**
When the switch is on and Capi is serving a logo this change enables a logo to appear on tag pages.

The logo will need some more styling!

image

/cc @steppenwells @Calanthe

### CODE REVIEW

File: common/app/common/commercial/BrandHunter.scala [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+import common.Edition
+import model.{Branding, Tags}
+import org.joda.time.DateTime
+
+object BrandHunter {
+
+  def findBranding(tags: Tags, publicationDate: Option[DateTime], edition: Edition): Option[Branding] = {
+
+    def findBrandingBySection(): Option[Branding] = None
```
Comment (steppenwells) [inline common/app/common/commercial/BrandHunter.scala]: I assume this is a placeholder for future section work.

Comment (kelvin-chappell) [inline common/app/common/commercial/BrandHunter.scala]: Yes, that's right.

Comment (kelvin-chappell) [inline common/app/common/commercial/BrandHunter.scala]: I'll add a note to say that.

**Comment by steppenwells** [comment]
Looks ok to me, 👍 but you might want eyes on it from someone who knows frontend better

File: onward/app/controllers/MostViewedSocialController.scala [hunk 2]
```diff
@@ -36,7 +36,7 @@ object MostViewedSocialController extends Controller with ExecutionContexts {
           val dataId = s"trending-on-$socialContext"
           val componentId = Some(s"trending-on-$socialContext")
           val displayName = Some(s"trending on $socialContext")
-          val properties = FrontProperties(None, None, None, None, false, None)
+          val properties = FrontProperties(None, None, None, None, false, None, None)
```
Comment (JonNorman) [inline onward/app/controllers/MostViewedSocialController.scala]: Can we leverage the `FrontProperties.empty` here?

File: onward/app/controllers/SeriesController.scala [hunk 3]
```diff
@@ -85,7 +85,7 @@ object SeriesController extends Controller with Logging with Paging with Executi
     val dataId = "series"
     val componentId = Some("series")
     val displayName = Some(series.displayName)
-    val properties = FrontProperties(series.tag.metadata.description, None, None, None, false, None)
+    val properties = FrontProperties(series.tag.metadata.description, None, None, None, false, None, None)
```
Comment (JonNorman) [inline onward/app/controllers/SeriesController.scala]: Same as above: can we use `FrontProperties.empty` here?

**Comment by JonNorman** [comment]
Looks sensible to me (with minor comment above) :+1:

**Comment by uplne** [comment]
I've pushed the change to fix the styling so it is beautiful now:

screen shot 2016-04-26 at 14 07 36
screen shot 2016-04-26 at 14 07 49

**Comment by kelvin-chappell** [comment]
Thanks @uplne

**Pull request merged**